Makefile.am

```diff
@@ -232,7 +248,22 @@ cpp_server_ct_server_SOURCES = \
 	cpp/server/certificate_handler.cc \
 	cpp/server/handler.cc \
 	cpp/server/json_output.cc \
-        cpp/server/log_processes.cc \
+  cpp/server/log_processes.cc \
```

Comment:
The indentation seems to be a bit strange, looking at it on GitHub... I'm suspecting spaces-vs-tabs? Remember that leading tabs are significant for `make`!